src/NamespaceManager.php

```diff
@@ -22,9 +22,9 @@ class NamespaceManager {
 	const DEFAULT_NAMESPACEINDEX = 100;
 
 	/**
-	 * @var LocalLanguage
+	 * @var LocalLanguage|null
 	 */
-	private $LocalLanguage;
+	private $localLanguage;
```

Comment:
There is a case where this is `null` (https://github.com/SemanticMediaWiki/SemanticMediaWiki/actions/runs/3866345626/jobs/6590412554#step:10:26).

This should be replaced with `?LocalLanguage`, but if we're still supporting PHP 7.3 in 4.0.x then this must wait a little bit.